src/renderer/components/publication/Cover.tsx

```diff
@@ -60,6 +60,8 @@ class Cover extends React.Component<IProps, null> {
                     className={styles.cover_img}
                     onClick={this.props.onClick}
                     onKeyPress={this.props.onKeyPress}
+                    role="presentation"
+                    alt="cover image"
```

Comment:
This needs translating. I'll merge this PR as-is, and I'll file a separate issue (so that work can progress on other important issues)

Reply:
https://github.com/readium/readium-desktop/issues/833